scripts/merge-dts.js

```diff
@@ -21,8 +21,8 @@ export default (packageDirectory, skipBundle = false) => {
   const dtsPath = join(packageDirectory, 'dist', 'dts');
 
   if (!skipBundle) {
-    const pkg = JSON.parse(readFileSync(join(packageDirectory, 'package.json')));
-    const dtsOutFile = join(packageDirectory, pkg.types);
+    const pkg = getPackageInfo(packageDirectory);
```

Comment:
I think it is better to split this on steps instead of extracting this logic into util function